tv_apps/smart-home/js/home.js

```diff
@@ -696,18 +853,29 @@
       this.clock.start(this.updateClock.bind(this));
     },
 
-    onEditFolder: function() {
-      this._cardPicker.show(this.cardScrollable.currentItem);
-    },
-
     get focusElem() {
       return this._focus;
     },
 
     get focusScrollable() {
       return this._focusScrollable;
+    },
+
+    get mode() {
+      return this.mainSection.dataset.mode;
+    },
+
+    set mode(newMode) {
+      if (this.mainSection.dataset.mode != newMode) {
```

Comment:
This is because we have to handle mode changing to 'cardPicker' or 'filter' in the home.js as well.